docs/application/web/get-started/overview.md

```diff
@@ -18,13 +18,29 @@ When creating your first Tizen Web application, you can select from the followin
 
     A TV application is targeted to run on smart TVs, and can be optimized for larger screens and remote control.
 
--    [Samsung Smart TV Web application](tv/first-samsung-tv-app.md)
+-   [Samsung Smart TV Web application](tv/first-samsung-tv-app.md)
 
-     A TV application is targeted to run on Samsung smart TVs, and can be optimized for larger screens and remote control.
+    A TV application is targeted to run on Samsung smart TVs, and can be optimized for larger screens and remote control.
 
 -   [Mobile Web application](mobile/first-app.md)
 
     A mobile application is targeted to run on smart phones, and can take advantage of a wide variety of functionalities available on those devices.
 
 The first application examples show a simple application with a basic UI and minimal functionality. Get familiar with the example first, and
 afterwards you can use it as a basis to build more complex applications.
+
+# Get Started with Web Service Applications
```

Comment:
The overview page introduces contents in profile order: Wearable, TV, and Mobile. 
When I see screenshot your new documents, there are mobile screenhosts.
So...you have to put your new document as the last item of the list.

The heading 1 same as H1 tag is used for the page title.
Please DON"T USE heading1 for the section title.

Moreover, please see how this page is shown on the screen: 
https://github.com/Samsung/tizen-docs/blob/acafc3cb0088c78641b547a67f80dc61f3a9793a/docs/application/web/get-started/overview.md.

> NOTE
> Please click **File changed** > **...** > **View file** before submiting a PR.

Reply:
> Global:
> Kindly use Sentence case for all the headings, as per the [style guide](https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#add-spacing-to-headings)
> 
> Global: As per Tizen docs W of Web is usually taken as a noun this W is written as capital. Kindly change "web" to "Web" wherever applicable irrespective of heading case too.
> Reference: https://developer.tizen.org/search/menu/web
> 
> ## Get started with Web service applications

I told you Surbhi that you guys would change heading styles after  the reviewing PRs are merged.

Reply:
@journey2w  Thanks and noted.

Reply:
> The overview page introduces contents in profile order: Wearable, TV, and Mobile.
> When I see screenshot your new documents, there are mobile screenhosts.
> So...you have to put your new document as the last item of the list.
 
There's no screenshot on overview page. Hope you check again.

The familiar target mobile screenshot is added on second depth:
(Web service can run on mobile, tv, and wearable targets)
https://github.com/YoungsooChoi/tizen-docs/blob/master/docs/application/web/get-started/web-service/first-service.md

Reply:
> The heading 1 same as H1 tag is used for the page title.
> Please DON"T USE heading1 for the section title.

I think it''s fixed by Web Application Addon:
https://github.com/Samsung/tizen-docs/blob/master/docs/application/web/get-started/overview.md